TelCo.ColorCoder/ColorMapper.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    class ColorMapper
```

Comment:
This class is called `ColorMapper`. But it's not mapping anything to anything else?